containers-tests/tests/intset-properties.hs

```diff
@@ -337,6 +337,10 @@ prop_foldL' s = foldl' (flip (:)) [] s == List.foldl' (flip (:)) [] (toList s)
 prop_map :: IntSet -> Bool
 prop_map s = map id s == s
 
+prop_mapMonotonic :: IntSet -> Bool
+prop_mapMonotonic s = mapMonotonic f s == map f s
+  where f = (* 7)
+
```

Comment:
This test is insufficient. Please generate arbitrary monotonic functions.

Reply:
Thanks for the quick response. I've updated the patch as requested. =
However I noticed that Data.Set doesn't have sufficient test either: https://github.com/haskell/containers/blob/14c46116a8b757907f0b56dbb3aee3712ea1d63f/containers-tests/tests/set-properties.hs#L539-L540. Would you mind if I add some similar tests to that in this PR as well?